src/ClientData/include/ClientData/CaptureDataHolder.h

```diff
@@ -38,6 +38,10 @@ class CaptureDataHolder {
     ORBIT_CHECK(HasCaptureData());
     return *capture_data_;
   }
+  [[nodiscard]] const CaptureData* GetCaptureDataPointer() const {
```

Comment:
This is needed, as we need to store a pointer of capture_data in the call tree view.